### DISCUSSION TITLE
For TV media, there should be an option for reset / retry media at all levels,

### DISCUSSION THREAD

**Discussion by legionsystems** [rivenmedia/riven-frontend#160] [Ideas]
Sometimes a wrong episode will download for a tv series, or you want to kick of a retry for an episode to get the latest version that's been released. (or maybe you want a season pack of a tv series if there is one)

So when viewing a tv series, the buttons Retry and Reset should be visible at the Series Level, the Season level and on each episode itself.
Apparently the backend already supports this.

![image](https://github.com/user-attachments/assets/a14fd104-44c1-45f2-b2ee-56b41e320f36)

**Comment by dreulavelle** [comment]
backend supports it, we just need the id! @AyushSehrawat

**Comment by AyushSehrawat** [comment]
You can see the options now!
